### ISSUE TITLE
Add vernacular names in multiple languages

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#108]
Camtrap DP is now used as export format in Agouti and users have asked if they can have the **vernacular names in other languages than English**. Camtrap DP currently supports that, because the definition says `preferably in English`. Since it only supports one language at a time, users could choose what language they want in their export. However, this makes the exports less portable (i.e. need one for local use and one for archiving).

It would be better if vernacular names in multiple languages could be expressed in the export. Not in `observations.csv`, as adding more fields would not be scalable, but in the metadata.

Currently that is:

```json
{
  "taxon_id": "11909338",
  "scientific_name": "Anas platyrhynchos",
  "vernacular_name": "mallard"
},
{
  "taxon_id": "11910101",
  "scientific_name": "Gallinula chloropus",
  "vernacular_name": "common moorhen"
},
```

But maybe we can make that:

```json
{
  "taxon_id": "11909338",
  "scientific_name": "Anas platyrhynchos",
  "vernacular_names": [
    {
      "language": "en",
      "name": "mallard"
    },
    {
      "language": "nl",
      "name": "wilde eend"
    }
  ]
},
{
  "taxon_id": "11910101",
  "scientific_name": "Gallinula chloropus",
  "vernacular_names": [
    {
      "language": "en",
      "name": "common moorhen"
    },
    {
      "language": "nl",
      "name": "waterhoen"
    }
  ]
},
```

That would also allow users (e.g. via package) to read these data and associate it in the language they want to the `observations.csv`

**Comment by peterdesmet** [comment]
With this setup, we can even question the need of a `vernacular_name` field in `observations.csv`.

**Comment by yliefting** [comment]
I'm in favor of this change. The proposed change would allow scaling to (potentially) dozens of languages, which I see as a nice addition to the format.

**Comment by kbubnicki** [comment]
Yes, this is a very good proposal! I also like the idea to remove `vernacular_name` from `observations.csv` when this proposal is implemented.

**Comment by peterdesmet** [comment]
Resolved in #121
